lib/elixir_google_scraper_web/templates/shared/_guest_user_navbar.html.eex

```diff
@@ -0,0 +1,3 @@
+<nav class="navbar fixed-top navbar-expand navbar-dark bg-dark">
+  <a class="navbar-brand" href="/">Google Scraper App</a>
```

Comment:
Should the logged in user see this nav as well 🤔

Reply:
Logged in user see also similar this nav, I implemented the logged in nav in https://github.com/junan/elixir_google_scraper/pull/30

So guest user nav is like
![Screen Shot 2021-06-04 at 4 07 00 PM](https://user-images.githubusercontent.com/6076762/120777256-496e8480-c54f-11eb-830a-1d8adeba892c.png)

 and logged in user nav is like
 
![Screen Shot 2021-06-04 at 4 06 47 PM](https://user-images.githubusercontent.com/6076762/120777323-5d19eb00-c54f-11eb-8590-f0be771df31c.png)